frontend/img/evy-e.svg

```diff
@@ -0,0 +1,127 @@
+<svg onload="initEvySVG(evt)" width="482" height="508" viewBox="0 0 241 254" fill="none" xmlns="http://www.w3.org/2000/svg">
+	<style>
+      path {
+        transition-property: stroke, stroke-width;
+      }
+      path.animate,
+      svg:hover path.animate {
+        transition-duration: 0.2s;
+        stroke-width: 12;
+      }
+      path:not(.animate) {
+        transition:
+          stroke-width 0.1s,
+          stroke 1s;
+      }
+      path:hover,
+      path.highlight.animate,
+      svg:hover path.highlight.animate {
+        transition-duration: 0.05s;
+        stroke-width: 13;
+      }
+      svg:hover path {
+        animation-duration: 0.7s;
+        animation-name: blip;
+      }
+      @keyframes blip {
+        0% {
+          stroke-width: 12;
+        }
+        30% {
+          stroke-width: 13;
+        }
+        100% {
+          stroke-width: 12;
+        }
+      }
+      @media not (prefers-reduced-motion) {
+        path.highlight.animate,
+        path:hover:not(.animate) {
+          stroke: #ffcf00;
+          stroke-width: 13;
+        }
+      }
+    </style>
+    <script>//<![CDATA[
+      function initEvySVG(e) {
+      	const svg = e.target.ownerDocument;
```

Comment:
nit: This line has a tab before `const` instead of 2 spaces. the rest of the lines are all space-indented.